.github/workflows/enable-egress.yml

```diff
@@ -11,6 +11,10 @@ on:
         description: 'Cloud Foundry Space'
         required: true
 
+env:
+  DATAGOV_DIR: datagov
```

Comment:
Where is this directory?  I don't see it in main branch nor being added here 😕 Is this specifying the name of the folder that the repo gets cloned to?

Reply:
It is used as the path for the [datagov repo](https://github.com/GSA/data.gov/blob/de441e6b2d486410a5e3ce47686e98fb0e38836b/.github/workflows/enable-egress.yml#L30), same as [CG_DIR](https://github.com/GSA/data.gov/blob/de441e6b2d486410a5e3ce47686e98fb0e38836b/.github/workflows/enable-egress.yml#L35). They are needed to provide a path so the script is able to reach into the other dir/repo.